README.md

```diff
@@ -1,153 +1,96 @@
 
-# jupyter_ascending
+# Jupyter Ascending
 
 Sync Jupyter Notebooks from any editor
 
 ![Jupyter Ascending](./media/simple_jupyter_ascending.gif)
 
 ## About
```

Comment:
Better description / to replace the about section:

Jupyter Ascending lets you edit Jupyter notebooks from your favorite editor, then instantly sync and execute that code in the Jupyter notebook running in your browser.

It's the best of both worlds--the autocomplete, keybindings, and refactoring tools you love in your favorite editor, plus the great visualization abilities of a Jupyter notebook.

Combined with basic syncing of your code to a remote server, you can have all the power of a beefy dev-server with all the convenience of editing code locally.